s3transfer/manager.py

```diff
@@ -27,7 +28,9 @@ def __init__(self,
                  multipart_threshold=8 * MB,
                  max_concurrency=10,
                  multipart_chunksize=8 * MB,
-                 max_queue_size=0):
+                 max_queue_size=0,
```

Comment:
I have not experimented. I just used around a similar ballpark of what boto3 was at (16 KB) since it shares a fair amount of the code. With the CLI though, the default could result in a max use of 1GB (1000 * 1MB ). That seems large to me. Maybe we make the default io queue 100 with 1MB chunk size?

One issue that I can think off of the bat with 1MB is that it decreases the amount of progress callbacks, giving you not as a fine-grain measurement of progress. So I can see why 16 KB was used in boto3.

What's your thoughts on good sizes? I think a smaller stream size is better due to the better callback granularity.